template/FeatherHttpTemplate.nuspec

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package>
+    <metadata>
+        <id>FeatherHttpTemplate</id>
```

Comment:
FeatherHttp.Templates

Reply:
No problem, would you like all instances of template to be pluralized in this pr? I thought you only needed name in template.json that way, sorry for the confusion 😅